api/src/main/java/com/codeforcommunity/rest/subrouter/SiteRouter.java

```diff
@@ -129,6 +133,34 @@ private void handleDeleteStewardshipRoute(RoutingContext ctx) {
     end(ctx.response(), 200);
   }
 
+  private void registerAddSite(Router router) {
+    Route addSiteRoute = router.post("/add");
+    addSiteRoute.handler(this::handleAddSiteRoute);
+  }
+
+  private void handleAddSiteRoute(RoutingContext ctx) {
+    JWTData userData = ctx.get("jwt_data");
+
+    AddSiteRequest addSiteRequest = RestFunctions.getJsonBodyAsClass(ctx, AddSiteRequest.class);
+
+    processor.addSite(userData, addSiteRequest);
+
+    end(ctx.response(), 200);
```

Comment:
❓ Should this be `201 CREATED` instead of `200 OK`?

Reply:
We've done `200 OK` for all of these types of returns, would need to be a separate ticket to switch them all